### PR TITLE
Add more HostSubnet logging

### PR DESCRIPTION
**Pull request by dcbw**
We don't expect nodes to be added/removed very often so the log
spam won't be large.

### CODE REVIEW

File: plugins/osdn/common.go [hunk 1]
```diff
@@ -51,6 +51,8 @@ type OsdnController struct {
 // Called by plug factory functions to initialize the generic plugin instance
 func (oc *OsdnController) BaseInit(registry *Registry, pluginHooks PluginHooks, multitenant bool, hostname string, selfIP string) error {
 
+	log.Infof("Starting with configured hostname '%s' (IP '%s')", hostname, selfIP)
+
```
Comment (pravisankar) [inline plugins/osdn/common.go]: We are already logging hostname/selfIP in https://github.com/dcbw/openshift-sdn/blob/more-hostsubnet-logging/plugins/osdn/common.go#L77-L79

Comment (dcbw) [inline plugins/osdn/common.go]: Yeah, but that's after the fixups.  I'd like to see what the configured/command-line values are too.

File: plugins/osdn/subnets.go [hunk 2]
```diff
@@ -49,7 +51,12 @@ func (oc *OsdnController) addNode(nodeName string, nodeIP string) error {
 		return fmt.Errorf("Error writing subnet %v to etcd for node %s: %v", sn, nodeName, err)
 	}
 
-	log.Infof("Created HostSubnet %v for node %s (%s)", sn, nodeName, nodeIP)
+	sub, err := oc.Registry.GetSubnet(nodeName)
+	if err != nil {
+		return fmt.Errorf("Error fetching subnet for node %q: %v", nodeName, err)
+	}
```
Comment (pravisankar) [inline plugins/osdn/subnets.go]: Unnecessary etcd call just for logging. Instead we could make CreateSubnet() to return the subnet along with the error (currently we are ignoring the returned subnet: https://github.com/dcbw/openshift-sdn/blob/more-hostsubnet-logging/plugins/osdn/registry.go#L105)

**Comment by pravisankar** [comment]
LGTM

**Pull request merged**